lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -1630,6 +1687,14 @@ impl FunctionCodeGenerator<CodegenError> for X64FunctionCode {
 
     fn begin_body(&mut self, _module_info: &ModuleInfo) -> Result<(), CodegenError> {
         let a = self.assembler.as_mut().unwrap();
+        let start_label = a.get_label();
+        // skip the patchpoint during normal execution
+        a.emit_jmp(Condition::None, start_label);
+        // patchpoint of 32 bytes
```

Comment:
Proposed text: " // Patchpoint of 32 1-byte nops."

Nops come in different lengths, fewer longer nops is faster in-place, but when replacing you have to replace whole instruction sequences at a time.